masonry/src/widget/sized_box.rs

```diff
@@ -512,5 +554,131 @@ mod tests {
         assert_render_snapshot!(harness, "empty_box_with_gradient_background");
     }
 
+    #[test]
+    fn label_box_with_padding_and_background() {
+        let widget = SizedBox::new(Label::new("hello"))
+            .width(40.0)
+            .height(40.0)
+            .background(Color::PLUM)
+            .border(Color::LIGHT_SKY_BLUE, 5.)
+            .padding(100.);
+
+        let mut harness = TestHarness::create(widget);
+
+        assert_debug_snapshot!(harness.root_widget());
+        assert_render_snapshot!(harness, "label_box_with_background_and_padding");
+    }
+
+    #[test]
+    fn label_box_with_padding_outside() {
+        let widget = SizedBox::new(
+            SizedBox::new(Label::new("hello"))
+                .width(40.0)
+                .height(40.0)
+                .background(Color::PLUM)
+                .border(Color::LIGHT_SKY_BLUE, 5.),
+        )
+        .padding(100.);
+
+        let mut harness = TestHarness::create(widget);
+
+        assert_debug_snapshot!(harness.root_widget());
+        assert_render_snapshot!(harness, "label_box_with_outer_padding");
+    }
+
     // TODO - add screenshot tests for different brush types
 }
+
+// --- MARK: PADDING STRUCT ---
```

Comment:
This definitely shouldn't be after the tests. The usual style in Masonry would have the definition of `Padding` just before the definition of the `SizedBox` type, then these impls before the `impl SizedBox` section.

Reply:
I moved it in the new commit.